src/test/lsipii/User/StatusInfo/Write.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+from enum import Enum
+
+from converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class StatusInfoValue(str, Enum):
+    SENT = ("SENT",)
+    PROCESSING = ("PROCESSING",)
+    WAITING_FOR_COMPLETION = ("WAITING_FOR_COMPLETION",)
+    READY = ("READY",)
+
+
+class StatusInfoResponse(CamelCaseModel):
+    status_label: str = Field(
+        ...,
+        title="Label of the status",
+        description="Label of the status in english",
+        example="Sent",
+    )
+    status_name: str = Field(
+        ...,
+        title="Name of the status",
+        description="Programmic identity field of the status",
```

Comment:
Same suggestion as in the other definition:
```suggestion
        description="Programmatic identity field of the status",
```